docs/get-started/quick-start-guide.md

```diff
@@ -60,84 +60,70 @@ Initialized
 
 ```console
 planet auth value
 ```
 
 Many `planet` calls you make require an API key. This is a very convenient way to quickly grab your API key.
 
-## Step 5: Creating your first order
-
-[*****TBW*****]
-
-The Planet SDK for Python allows Python developers to write software that makes
-use of the following Planet APIs:
-
-* [orders](https://developers.planet.com/docs/orders/)
-* [data](https://developers.planet.com/docs/data/)
-* [subscriptions](https://developers.planet.com/docs/subscriptions/)
-* [basemaps](https://developers.planet.com/docs/basemaps/), [tasking](https://developers.planet.com/docs/tasking/) and [analytics](https://developers.planet.com/docs/analytics/) API's are not yet implemented.
-
-The client modules within the Python library are asynchronous, which greatly
-speeds up many interactions with Planet’s APIs. Support for asynchronous
-development is native to Python 3.6+ via the
-[`asyncio` module](https://docs.python.org/3/library/asyncio.html). A great
-resource for getting started with asynchronous programming in Python is
-https://project-awesome.org/timofurrer/awesome-asyncio. The Writings and Talks
-sections are particularly helpful in getting oriented.
-
-Let’s start with creating an order with the Orders API:
-
-```python
->>> import asyncio
->>> import os
->>> import planet
->>>
->>> request = {
-...   "name": "test_order",
-...   "products": [
-...     {
-...       "item_ids": [
-...         "3949357_1454705_2020-12-01_241c"
-...       ],
-...       "item_type": "PSOrthoTile",
-...       "product_bundle": "analytic"
-...     }
-...   ]
-... }
-...
->>> async def create_order(request):
-...     async with planet.Session() as ps:
-...         client = planet.OrdersClient(ps)
-...         return await client.create_order(request)
-...
->>> oid = asyncio.run(create_order(request))
+#### Your API Key as an Environment Variable
 
+You can also set the value of your API Key as an environment variable in your terminal at the command line:
+
+```console
+export PL_API_KEY=<your api key>
+```
+
+And you can see that the value was stored successfully as an environment variable with the following command:
+
+```console
+echo $PL_API_KEY
 ```
 
-Not into async? No problem. Just wrap the library and async operations together
-and call from your synchronous code.
+!!!note "Planet SDK uses the API Key environment variable"
+    If you do create a `PL_API_KEY` environment variable, the SDK will use this value. `PL_API_KEY` overrides the value that was retrieved from the Planet server with a call to `planet auth value`. The initial `planet auth value` won’t be updated to reflect the new value.
 
-```python
->>> def sync_create_order(order_details):
-...     return asyncio.run(create_order(order_details))
->>>
->>> oid = sync_create_order(order_details)
+## Step 5: Search for Planet Imagery
 
+You’ve installed the environment, the SDK, and connected with the Planet server. You’re now ready to get your first bunch of data.
```

Comment:
I don't love the language with connect/authenticate with 'the Planet server', since it's not just one service. A straightforward replacement that would make sense to me would be 'Planet's auth service', but could also just talk about like 'authenticating' and/or 'registering' your account. 

I can make this its own issue if you want to discuss more, as I'm happy for this to merge as is. But we'd ideally update all the mentions of 'planet server'.

Reply:
changed to "connected to Planet services"